### ISSUE TITLE
Spacing layout missing

### ISSUE THREAD

**Issue by vidasci** [tafcoder/sleek-dashboard#307]
Hello, 

In the demo-version ([demo](https://sleek.tafcoder.com/index.html)) there is a spacing-layout option in the layout settings. However, I've noticed this is not available in the latest release. Can you tell me how I can implement this spacing layout option in a project? 

Thnx!

**Comment by envatic** [comment]
Yeah it seems the demo and Repo are different, several items in the demo , not in the repo,  I ended up having to rip the demo
